2025.md

```diff
@@ -0,0 +1,3 @@
+## ISQCMC ’25
+
+- [conference website](https://quantummusicsymposium.wordpress.com/)
```

Comment:
Just to note/remind that this domain is likely changing, but it does not compromise this pull request - which is only including the upcoming event to the website